perl.yaml

```diff
@@ -78,6 +78,18 @@ pipeline:
   - uses: strip
 
 subpackages:
+  - name: miniperl
+    description: "miniperl"
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}"/usr/bin
```

Comment:
```suggestion
          mkdir -p "${{targets.subpkgdir}}"/usr/bin
```

Reply:
Ridiculous, I wasted far too much time building this locally and failing, thanks for catching!